### ISSUE TITLE
enable shenandoah during build of jdk11

### ISSUE THREAD

**Issue by judovana** [adoptium/installer#1014]
On jdk11 shenandaoh is present for jdk11 on x64 and  aarch64, it was safely built on fedora for years. Maybe would be nice to enable it.

**Comment by jiekang** [comment]
Shenandoah was enabled for 11.0.9 onwards, is it not available in the installers? 

Relevant blog post:
https://blog.adoptopenjdk.net/2020/10/adoptopenjdk-8u272-1109-and-1501-available/

Relevant code:
https://github.com/adoptium/temurin-build/blob/master/sbin/build.sh#L114

**Comment by karianna** [comment]
@judovana where is this missing?

**Comment by judovana** [comment]
It is enabled only for "temurin" build variant. Which is  "private" - not mentioned in readmes, helps and so on (they offer only hotspot or openj9). In hotspot, it is unluckily disabled. Double unluckily it is disbaled also in upstream jdk11  (https://bugs.openjdk.org/browse/JDK-8342631, maybe others).  

I will be repacking temurins, so from this scope, it is ok.  (Actually I filled this bug before I found that next to "hotspot" (default) and "openje9" variants is also  "temurin"

In addition,  I will be using temurin-build/make-anyu-jdk.sh to build system jdk and future system jdk (altest sts). I know those jdks already have shenandoah enabled, but as for now, I'm not sure if I will be using "hotspot" or "temuirn" varint.  I would welcome feedback on that part.

So form scope of https://github.com/adoptium/installer/issues/848 it is done. ( I will renmvoe it from https://github.com/adoptium/installer/issues/848#issuecomment-2133516101 )

The  question whether to enable it for "hotpsot" variant remains opened. I vote for enabling it.  Thanx both!

**Comment by jiekang** [comment]
I don't agree with enabling it for the "hotspot" variant. That represents the upstream which as you've noted, has disabled it. Any distribution that enables it is diverging from upstream and that is understood via a different variant.

Regarding READMEs, that's a good point of feedback. I don't think there was a thorough updating of documentation when the "temurin" variant was introduced. We should look into that.

**Comment by judovana** [comment]
Sure thing. TY. In meantime, I had added: https://github.com/adoptium/temurin-build/pull/4019/files#diff-35aa562c213f5458cbd0783c26dc14225598ff301ed98119a8858f1814bad836R73 which is partially it. But generally the diifferencwes between  temurin/hotspot build variants may be more easy to find. Thanx a lot!
